thoth/user_api/configuration.py

```diff
@@ -34,6 +34,7 @@ class Configuration:
     THOTH_BACKEND_NAMESPACE = os.environ["THOTH_BACKEND_NAMESPACE"]
     THOTH_DEPLOYMENT_NAME = os.environ["THOTH_DEPLOYMENT_NAME"]
     THOTH_HOST = os.environ["THOTH_HOST"]
+    INTERNAL_SECRET = os.getenv("THOTH_INTERNAL_API_SECRET", "secret101")
```

Comment:
```suggestion
    INTERNAL_SECRET = os.getenv("THOTH_INTERNAL_API_SECRET")
```

Please remove the secret from sources.

Reply:
> Let's default to always set secret to some sane default not to potentially introduce a security hole

Is this not what you meant by this comment?

Reply:
No password should be present in sources. Please add the default encrypted password to thoth-application where this configuration should live.